chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -493,6 +505,7 @@ save.link.error					= An error occurred while saving this link.
 save.error.cannotMakeChangesToCollection = You cannot make changes to the currently selected collection.
 save.error.cannotAddFilesToCollection = You cannot add files to the currently selected collection.
 save.error.cannotAddToMyPublications = You cannot save items directly to My Publications. To add items to My Publications, drag them from another library.
+save.error.cannotAddToFeed = You cannot save items directly to feeds.
```

Comment:
"You cannot save items to feeds." (no "directly" here because you can't add items indirectly either, whereas with My Publications you can actually drag items in)